packages/fxa-shared/db/models/auth/prune-tokens.ts

```diff
@@ -84,12 +84,22 @@ export class PruneTokens extends BaseAuthModel {
           '@sessionTokensDeleted',
         ]
       );
-      this.metrics?.increment('PruneTokens.Complete', { ...result });
+
+      if (result) {
+        Object.entries(result).forEach(([key, value]) => {
+          if (typeof value === 'number' && value > 0) {
+            this.metrics?.increment(
+              `prune-tokens.complete.${key.replace(/[^a-zA-Z0-9-]/, '')}`,
+              value
+            );
```

Comment:
Do we not know the list of keys ahead of time?

Reply:
We do, but if the script changes in the future, I was thinking it'd be nice to not have to also update this metric. I was thinking any value in the result with a count would be considered a valid metric.

Reply:
I can also the value of being explicit here. Thoughts?

Reply:
IIUC we don't need that `.replace`, since know those values are safe.

Reply:
There are @ symbols in the names, and that might be a problem. Hence the replace.

Reply:
I'll change it to be explicit. And make the string manipulation even better.

Reply:
More complicated but better maybe. This now makes names kebab case which seems to be the convention for metrics.